crates/matrix-sdk/src/room/timeline/event_handler.rs

```diff
@@ -773,18 +773,16 @@ fn _update_timeline_item(
     }
 }
 
-/// Converts a timestamp since Unix Epoch to a local date and time.
-fn timestamp_to_local_datetime(ts: MilliSecondsSinceUnixEpoch) -> DateTime<Local> {
-    Local
+/// Converts a timestamp since Unix Epoch to a year, month and day.
+fn timestamp_to_ymd(ts: MilliSecondsSinceUnixEpoch) -> (i32, u32, u32) {
```

Comment:
I would prefer to get a:

```rust
struct Ymd {
    pub y: i32,
    pub m: u32,
    pub d: u32,
}
```

type here. Otherwise, it's quite easy to misunderstand the tuple, don't you think?

Reply:
Or at least, a type alias: `type Ymd = (i32, u32, u32)`, at least we have the name which can help to find the meaning of the tuple, but I wouldn't favor this solution.

Reply:
We never access the individual fields directly. But why not.